pkg/build/builder/docker_test.go

```diff
@@ -0,0 +1,87 @@
+package builder
+
+import (
+	"testing"
+)
+
+func TestReplaceValidCmd(t *testing.T) {
```

Comment:
That should suffice IMHO, just running `Parse` on the new file should be a good start without any erros is OK.